src/util/scale.js

```diff
@@ -0,0 +1,16 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.util.scale = {
+        // ordinal axes have a rangeExtent function, this adds any padding that
+        // was applied to the range. This functions returns the rangeExtent
+        // if present, or range otherwise
+        range: function(scale) {
+            return scale.rangeExtent ? scale.rangeExtent() : scale.range();
+        },
+
+        isOrdinal: function(scale) {
+            return scale.rangeBands;
```

Comment:
Good spot - this was based on a refactor of some D3 code, which doesn't appear to be self-consistent regarding how it identified ordinal axes.